src/app/compliance-rule-configuration/compliance-rule-configuration.component.html

```diff
@@ -0,0 +1,45 @@
+<div>
+  <mat-form-field appearance="fill" >
+    <mat-label>Create new Configuration from Rule:</mat-label>
+    <mat-select [(value)]="selectedRule">
+      <mat-option *ngFor="let complianceRule of complianceRules" [value]="this.utils.getId(complianceRule)">
+        Rule {{this.utils.getId(complianceRule)}}
+      </mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <br>
+  IssueType:
```

Comment:
I think, from a UX point of view, it is better to have homogeneous handling of such situations.
I am referring to the case when we have a parent entity connected to child entities. Possible "types" of child entities are presented in a combobox, you select one and click Add. This creates a dummy entity of the child type. You click Configure on this child entity, and then you are presented with a dialog to actually configure it. Then you can save the changes.
This is how we implemented the refinement plugins list associated with a compliance job. So I guess we should do it similarly here:
Let's not require the user to specify the issue type just yet, and instead use a blank string. Then in the configuration dialog, we allow the user to specify it.
What do you think?

Reply:
I need the issueType to present possible fixing plugins however. I would just let the user do it for now